### PR TITLE
[BACKLOG-9318] - Create flat list selector component (functionally)

### PR DESCRIPTION
**Pull request by plagoa**
- Added base class to the stylings to prevent styling conflicts
- Fixed some styles

### CODE REVIEW

**Comment by plagoa** [comment]
@marcoslarsen @candentira please review

File: src/app/tab/tab.css [hunk 1]
```diff
@@ -33,80 +34,80 @@ body, p, ul {
...
+  width: 250px;
   height: 30px;
   position: absolute;
   top: -100px;
   right: 0;
   z-index: 2;
 }
 
-.drag-field {
+.components-wrapper .drag-field {
```
Comment (candentira) [inline src/app/tab/tab.css]: in dragstart handler drag-field is added to body root, not inside components-wrapper

Comment (plagoa) [inline src/app/tab/tab.css]: You're right, let me fix that.

**Comment by plagoa** [comment]
@candentira fixed the css issue. please review.

**Pull request merged**